src/main/scala/cognite/spark/v1/PushdownUtilities.scala

```diff
@@ -154,15 +162,11 @@ object PushdownUtilities {
     }
 
   def externalIdsSeqFromWrappedArray(wrappedArray: String): Seq[String] =
-    // We get "WrappedArray(ext1, ext2)" here, so we can remove irrelevant parts of the string
-    // and extract ext1 and ext2 from it -> return Seq('ext1', 'ext2')
-    wrappedArray.length match {
-      case 14 => Seq() // "WrappedArray()".length
-      case _ => wrappedArray.slice(13, wrappedArray.length - 1).split(',')
-    }
+    // we get a string separated by \0
+    wrappedArray.split('\u0000')
 
   def idsFromWrappedArray(wrappedArray: String): Seq[Long] =
-    wrappedArray.split("\\D+").filter(_.nonEmpty).map(_.toLong)
+    wrappedArray.split('\u0000').map(_.toLong)
```

Comment:
I don't know why we needed `.filter(_.nonEmpty)` before, but that also means I don't know why we can remove it now. ;)

Reply:
Ok, I'll add it back, but I also have no idea why would it be needed.